zerver/webhooks/slack_incoming/tests.py

```diff
@@ -24,7 +24,6 @@ def test_message_formatting(self) -> None:
         tests = [
             ("some *foo* word", "some **foo** word"),
             ("*foo*", "**foo**"),
-            ("*foo* *bar*", "**foo** **bar**"),
```

Comment:
This is has been moved to `slack_message_conversion.json` fixture as `unsupported_bold_test`. The difference is, instead of space, `unsupported_bold_test` use `\n` which is the more probable scenario.